Modules/audioop.c

```diff
@@ -5,12 +5,8 @@
 
 #include "Python.h"
 
-#if defined(__CHAR_UNSIGNED__)
-#if defined(signed)
 /* This module currently does not work on systems where only unsigned
    characters are available.  Take it out of Setup.  Sorry. */
```

Comment:
The last part of the comment should be removed, it is confusing without the preprocessor guards. I'd consider just dropping the comment, a compiler where "signed char" is not signed should be considered broken.

Reply:
The comment was introduced by b6775db24 in 1994, just five years after the first ever C standard was published. Internet distribution was slow and software companies were rigid back then so no surprise compatibility with per vendor compiler quirks was a real deal.

Reply:
Please remove this outdated comment. audioop explicitly uses ``signed char`` and ``unsigned char``. Maybe to avoid any risk of confusion, the C code should use int8_t and uint8_t.

Reply:
The comment is pining for the fjords.